webapp/app.py

```diff
@@ -527,16 +527,16 @@ async def gzip_middleware(request, handler, **kwargs):
         res = await handler(request, **kwargs)
         header = 'Accept-Encoding'
 
-        if not request.path.startswith('/api/v'):  # exception not to gzip swagger pages
+        try:
+            if res.body is not None and header in request.headers and "gzip" in request.headers[header]:
```

Comment:
Nothing is returned if condition on L531 is false. There are two returns in gzip_middleware function, I would keep one return at the end.

Reply:
fixed